### PR TITLE
2019 proposed updates

### PR DESCRIPTION
**Pull request by jcarbaugh**

### CODE REVIEW

**Review by alexbarbato — APPROVED** [review]
Looks great to me! Thank you!

**Comment by jcarbaugh** [comment]
Will hold on merging per pending discussion with @CorneliusIV

**Comment by alexbarbato** [comment]
As an aside - I might propose removing the sass static_src as it's not imported nor is it compilable at the moment (maybe replace with a generic css file?) and import the app.js file in the base template (or remove?)

**Comment by jcarbaugh** [comment]
@alexbarbato I made those same changes locally after generating the project for hotlinks, so I definitely need to merge that back here. Thanks for pointing that out!

File: {{cookiecutter.repo_name}}/mo.yml [hunk 1]
```diff
@@ -1,4 +0,0 @@
-domain: {{ cookiecutter.domain_name }}
```
Comment (CorneliusIV) [inline {{cookiecutter.repo_name}}/mo.yml:1]: Do we need this file for the command line tool?

**Comment by CorneliusIV** [comment]
We can probably also remove:
- .babelrc
- .nvmrc
- Any reference to the node.js build packs in the Readme and app.json

**Comment by CorneliusIV** [comment]
Do we want to add the webpack-loader this round?

**Pull request closed without merging**